include/exadg/fluid_structure_interaction/driver.cpp

```diff
@@ -231,9 +231,13 @@ Driver<dim, Number>::coupling_structure_to_fluid(bool const extrapolate) const
   VectorType velocity_structure;
   structure->pde_operator->initialize_dof_vector(velocity_structure);
   if(extrapolate)
+  {
```

Comment:
I will pass iteration and use `get_velocity_n()` instead of `..._np()` to not assume the velocity from t^n still be stored in the vector for t^{n+1}